### PR TITLE
Downgrade conda-build to 1.20.0

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://github.com/conda-forge/staged-recipes/issues/448#issuecomment-214243144

### CODE REVIEW

**Comment by ocefpaf** [comment]
@pelson and @jakirkham is that OK for now?

**Comment by pelson** [comment]
I agree with your sentiment in https://github.com/conda-forge/staged-recipes/issues/448#issuecomment-214243659, but we are on slower time for conda-smithy I think.

**Comment by jakirkham** [comment]
> @pelson and @jakirkham is that OK for now?

It is the only thing that will work until the cause is determined. @patricksnape said something that suggested to me he may know the cause of the problem. Maybe he can come up with an actual fix.

We really need to have an issue over there. When I suggested someone try this fix to a couple of people, I was on my phone waiting to catch a plane and have not looked at this since. Given you guys have decided this is the fix and we need it, I think the next step is to raise this upstream. Does anyone know if this issue actually got raised on conda-build?

It is worth noting that 1.20.1 was the release that made this workaround ( https://github.com/conda-forge/staged-recipes/pull/447 ) unnecessary. However, as we have gone back to 1.20.0, we now may need that workaround back. For now, I'm going to hold off on reverting that change because it is a bit tedious.

**Comment by jakirkham** [comment]
> However, as we have gone back to 1.20.0, we now may need that workaround back.

Actually, never mind, I retract that comment.

We shouldn't need to bring the workaround back as the pinning should only be needed for Python 3.4. This means that Python 2.7 builds should still proceed correctly as they can still use conda-build 1.20.1.

**Comment by jakirkham** [comment]
Thanks @ocefpaf. Sorry I forgot to say the most important thing.

File: appveyor.yml [hunk 1]
```diff
@@ -76,6 +76,9 @@ install:
     - cmd: conda config --add channels conda-forge
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: obvci_install_conda_build_tools.py
+    # Workaround for Python 3.4 and x64 bug in latest conda-build.
+    # FIXME: remove this once a new conda-build is released!
+    - cmd: conda install --yes conda-build=1.20.0
```
Comment (jakirkham) [inline appveyor.yml]: So, we shouldn't do this on all platforms or we need the old CI fixes back.

File: appveyor.yml [hunk 2]
```diff
@@ -76,6 +76,9 @@ install:
     - cmd: conda config --add channels conda-forge
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: obvci_install_conda_build_tools.py
+    # Workaround for Python 3.4 and x64 bug in latest conda-build.
+    # FIXME: remove this once a new conda-build is released!
```
Comment (jakirkham) [inline appveyor.yml]: There is no fix AFAICT. This is still very much an open issue.

**Comment by jakirkham** [comment]
See this PR ( https://github.com/conda-forge/staged-recipes/pull/455 ), which addresses the concerns noted above.

**Pull request merged**